ironic_tempest_plugin/tests/scenario/ironic_standalone/test_basic_ops.py

```diff
@@ -436,3 +436,18 @@ class BaremetalRedfishIPxeWholediskHttpLink(
     @utils.services('network')
     def test_ip_access_to_server(self):
         self.boot_and_verify_node()
+
+class BaremetalIdracDirectWholediskHttpLink(
```

Comment:
class name can be changed to BaremetalIdracVirtualMedia

Reply:
ohk

Reply:
Done